manim/mobject/graphing/probability.py

```diff
@@ -304,6 +304,19 @@ def __init__(
 
         self.y_axis.add_numbers()
 
+    def _update_colors(self):
+        """Initialize the colors of the bars of the chart.
+
+        Sets the color of ``self.bars`` via ``self.bar_colors``.
+
+        Primarily used when the bars are initialized with ``self._add_bars``
+        or updated via ``self.change_bar_values``.
+        """
+        if isinstance(self.bar_colors, str):
+            self.bars.set_color_by_gradient(self.bar_colors)
+        else:
+            self.bars.set_color_by_gradient(*self.bar_colors)
```

Comment:
Given that the type hint for `bar_colors` also supports `str` I think this is fine for that but since `bar_colors` is plural can we not just tell the user to always enter the values in a list? 
And remove this function entirely and just put in `self.bars.set_color_by_gradient(*self.bar_colors)` in place of where it is called?

Reply:
I like the suggestion of forcing `bar_colors` to be inputted as a list. but I think having colour updating in a separate function is clearer about conveying its purpose, even if it looks a little silly in its current state 🤷